### PR TITLE
Provide a method for adding custom LDAP CA cert

### PR DESCRIPTION
**Pull request by tjmoore4**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [x] New feature
 - [ ] Bug fix
 - [ ] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

This update allows a custom CA cert to be mounted for Postgres LDAP authentication. This uses the existing spec.config.files method to mount a Secret containing the ca.crt file. The required path and file name is 'ldap/ca.crt'.

**Other Information**:
Issue: PGO-1000

### CODE REVIEW

File: internal/patroni/config.go [hunk 1]
```diff
@@ -450,6 +450,19 @@ func instanceEnvironment(
 			Name:  "PATRONICTL_CONFIG_FILE",
 			Value: configDirectory,
 		},
+		// This allows a custom CA certificate to be mounted for Postgres LDAP
+		// authentication via spec.config.files.
+		// - https://wiki.postgresql.org/wiki/LDAP_Authentication_against_AD
+		//
+		// When TLS_CACERT is used as an environment variable rather 'LDAP' is
```
Comment (dsessler7) [inline internal/patroni/config.go:457]: Rather? Not sure I understand this wording.

Comment (andrewlecuyer) [reply internal/patroni/config.go:457]: ⛏️ This is the only things that jumped out at me as well.  And might be worth noting that the Environment Variables section of that doc does spell out the proper syntax for OpenLDAP env vars:

```
LDAP<option-name>
              Set <option-name> as from ldap.conf
```

**Review by andrewlecuyer — APPROVED** [review]

File: internal/patroni/config.go (hunk 1 above)
Comment (tjmoore4) [reply internal/patroni/config.go:457]: I'll work on rewording that comment, looks like I missed this during some refactoring.

Comment (tjmoore4) [reply internal/patroni/config.go]: Updated. Thank you both for the catch.

**Pull request merged**